tests/foreman/ui/test_ansible.py

```diff
@@ -115,3 +118,65 @@ def test_positive_create_variable_with_overrides(target_sat):
             }
         )
         assert session.ansiblevariables.search(key)[0]['Name'] == key
+
+
+@pytest.mark.skip_if_open("BZ:2155814")
+@pytest.mark.no_containers
+@pytest.mark.rhel_ver_list([8, 9])
```

Comment:
Is it possible to include EL7 and fips hosts?

Reply:
The Role which I am using is related to insights-client and that package is not present on rhel 7 so didn't include that. I need to explicitly install that on the VM then I think include that.

Reply:
@adarshdubey-star I guess that role is just an example here, so i wouldn't skip el7 just for that purpose, can you try using any other role instead where we can test all versions?